src/test/unit/manipulation-test-helper.js

```diff
@@ -124,6 +131,10 @@ function manipulationTestHelper(a) {
         executions.push(EXECUTE.UI_CLICK);
     }
 
+    if (typeof a.manipulationKeyCombo === "string") {
+        executions.push(EXECUTE.KEY_COMBO);
```

Comment:
I've moved the check for `skipKeyboardShortcutTests` from here to inside `manipulateAndAssert`, below. It fixed a bug.